middleware/gateway/source/group/inbound/main.cpp

```diff
@@ -82,11 +84,16 @@ namespace casual
                            state.alias = message.model.alias;
                            state.limit = message.model.limit;
 
+                           auto is_enabled = []( auto& connection){ return connection.enabled;};
+                           auto [ enabled, disabled] = algorithm::stable::partition( message.model.connections, is_enabled);
+
+                           state.disabled_connections = algorithm::container::vector::create( disabled);
+
```

Comment:
Ahh, right. The disabled are kept on the side, and if any of them are present, they will end upp in `change.removed`, and disconnect as before...